CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# 2.0.0
+
+## Other
+
+- [#22](https://github.com/okta/okta-ios-jwt/pull/22) - Updates Swift Version to `4.2` and adds custom `User-Agent` header to outgoing requests.
```

Comment:
Do we need to update CHANGELOG with 1.0.0 changes as well? Was the 0 version published?

Reply:
We went from `0.x` to `1.x` with no code changes, it was simply to indicate that this library is "officially supported".

Therefore, the CHANGELOG will not have any meaningful items.